### DISCUSSION TITLE
llama-parallel is processing only one request, when run with various parameters

### DISCUSSION THREAD

**Discussion by skprasadu** [ggml-org/llama.cpp#8388] [Q&A] [answered]
I cloned and built llama.cpp in my Ubuntu GPU A100 box. And why I am running this example as below

`./llama-parallel --prompt "where is bangalore\nwho is lord krishna" --parallel 2 --cont-batching --sequences 1 -ntg 1024 -npp 1024,1024 -ngl 35` I am only getting response for  1st request. The second request's response is not printing. Am I missing something. Can someone, put a readme of how we do parallel processing of prompts.

**Comment by skprasadu** [comment, marked as answer]
I think I figured out, this is the right command, 
`./llama-parallel --prompt "where is bangalore\nwho is lord krishna" --parallel 2 --cont-batching --sequences 2 -ntg 1024 -npp 1024,1024 -ngl 35` actually we need to give the sequence size equal to the number of prompts that need to be processed.

Hope it will be useful to others.

**Reply by ggerganov** [reply]
The `-ntg` and `-npp` args are ignored by the `llama-parallel` so you can remove those
